### ISSUE TITLE
Can't `want: false` in taptest

### ISSUE THREAD

**Issue by m-dango** [yaml/yamlscript#180]
Unsure if I'm doing something wrong but I seem to be unable to use `false` in a taptest check:

```yaml
#!/usr/bin/env ys-0

require ys::taptest: :all

use: 'leap'

plan: 1

test::
- name: year not divisible by 4 in common year
  eval: leap/is-leap-year(2015)
  want: false
```

```
1..1
Error: taptest: Test 1 requires one of: 'want', 'like', 'has'
```

**Comment by ingydotnet** [comment]
Yeah that's a bug. It will get fixed and released by tomorrow morning.

**Comment by ingydotnet** [comment]
Fixed the bug but still shoring up the test framework a bit.

**Comment by ingydotnet** [comment]
@m-dango left you info in our matrix chat.

**Comment by ingydotnet** [comment]
I released 0.1.67 which fixes things and more.

I'll write docs for `ys::taptest` soon but until then this test file can help you figure it out: https://github.com/yaml/yamlscript/blob/main/ys/test/taptest.t

**Comment by ingydotnet** [comment]
@m-dango The docs for `ys::taptest` are up here.

I'll be putting out a release with a few more changes today. I'll update the docs when I do.
